ethcore/sync/src/lib.rs

```diff
@@ -26,7 +26,6 @@ extern crate ethcore_network_devp2p as devp2p;
 extern crate parity_bytes as bytes;
 extern crate ethcore_io as io;
 extern crate ethcore_transaction as transaction;
-#[macro_use]
```

Comment:
Looks like this macro is used only in tests, so replacing this line with `#[cfg_attr(test, macro_use)] ` should do the trick.
see https://github.com/rust-lang/rust/issues/44342.

Reply:
Thanks :+1: